testRunner.js

```diff
@@ -130,6 +130,7 @@ function* check(file) {
                 result = result.split('|undefined|').join('x');
                 result = result.split('efault: undefined').join('x');
                 result = result.split('"undefined",').join('x');
+                result = result.split('undefined,').join('x');
```

Comment:
Hmm, I don't think this should be necessary given this change. Do you happen to know which input definition(s) caused the problem to show up?

Reply:
I believe it is: https://github.com/APIs-guru/openapi-directory/blob/master/APIs/amazonaws.com/apigateway/2015-07-09/swagger.yaml. 

Further, I believe it is because of [this line](https://github.com/APIs-guru/openapi-directory/blob/master/APIs/amazonaws.com/apigateway/2015-07-09/swagger.yaml#L6297). That description has "undefined" in it and is getting included because of my change.